notify/impl.go

```diff
@@ -658,6 +659,7 @@ func (n *OpsGenie) Notify(ctx context.Context, as ...*types.Alert) error {
 			opsGenieMessage: &apiMsg,
 			Message:         tmpl(n.conf.Description),
 			Details:         details,
+			Source:          tmpl(n.conf.Source),
```

Comment:
@brian-brazil it already is: https://github.com/prometheus/alertmanager/blob/master/config/notifiers.go#L88

Which is expanded to:
https://github.com/prometheus/alertmanager/blob/master/template/default.tmpl#L34
https://github.com/prometheus/alertmanager/blob/master/template/default.tmpl#L2